index.bs

```diff
@@ -11290,6 +11290,35 @@ defined below, skipping over those channels not present.
 		<tr><td>17 <td>SPEAKER_TOP_BACK_RIGHT <td> <td> <td> <td>
 </table>
 
+<h3 id="channels-tail-time">
+Implication of tail-time on input and output channel count</h3>
+
+When an {{AudioNode}} has a non-null <a>tail-time</a>, and an output channel
+count that depends on the input channels count, any change in input channel
+MUST take the {{AudioNode}}'s <a>tail-time</a> into account
+
+When there is a decrease in input channel count, the change in output channel
+count MUST happen after the <a>tail-time</a> has passed.
```

Comment:
I'm trying to think whether this is appropriate for DelayNode.  That would be fine if tail-time corresponds to the current `delayTime` in the node.  But perhaps "has any internal processing state which has not yet been emitted" corresponds more closely with `maxDelayTime`.  I assume we don't want to always wait `maxDelayTime`.

```suggestion
count MUST happen when the input that was received with greater channel count no longer affects the output.
```
which would still be fine for other nodes.

Reply:
I think it's impossible to talk in terms of tail time for this indeed, and it's best to be more generic.